tools/dotcms-cli/api-data-model/src/main/java/com/dotcms/common/LocationUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.dotcms.common;
+
+import com.dotcms.model.config.Workspace;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class LocationUtils {
+
+    private LocationUtils() {
+        //Hide public constructor
+    }
+
+    /**
+     * Checks if the given URL is a folder.
+     *
+     * @param url the URL to check
+     * @return true if the URL is a folder, false otherwise
+     */
+    public static boolean URLIsFolder(final String url) {
```

Comment:
I would rename these methods to something like IsFolderURL to stay consistent with java naming conventions